android-lib/src/androidTest/java/com/kinvey/androidTest/store/data/DataStoreMultiInsertTest.java

```diff
@@ -1,1980 +0,0 @@
-package com.kinvey.androidTest.store.data;
```

Comment:
I'm thinking if we should keep the unit tests in Java for now so we can assure the code is behaviour just like before the Kotlin migration

Reply:
@heyzooi +1. We should not migrate tests written in Java until we have completed the Kotlin migration and verify that all existing tests still pass.

Reply:
Reverted changes of old `DataStoreMultiInsertTest.java` tests file, keep new `DataStoreMultiInsertTest.kt` tests file but set it as `Ignored` when tests run.